client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-detail-view/motion-detail-view.component.ts

```diff
@@ -301,17 +320,36 @@ export class MotionDetailViewComponent extends BaseMeetingComponent implements O
     public setSurroundingMotions(): void {
         const indexOfCurrent = this._sortedMotions.findIndex(motion => motion === this.motion);
         if (indexOfCurrent > 0) {
-            this.previousMotion = this._sortedMotions[indexOfCurrent - 1];
+            this.previousMotion = this.findNextSuitableMotion(indexOfCurrent, -1);
         } else {
             this.previousMotion = null;
         }
         if (indexOfCurrent > -1 && indexOfCurrent < this._sortedMotions.length - 1) {
-            this.nextMotion = this._sortedMotions[indexOfCurrent + 1];
+            this.nextMotion = this.findNextSuitableMotion(indexOfCurrent, 1);
         } else {
             this.nextMotion = null;
         }
     }
 
+    /**
+     * Substitutes the previous or next motion.
+     * Fires if @var this._amendmentsInMainList as well as @var this._navigatedFromAmendmentList collide.
+     * @param indexOfCurrent The index from the active motion.
+     * @param previous Wether the method is called for the previous or the next motion.
+     */
```

Comment:
Doc needs to be updated or removed. For private methods it is normally okay to not add a docblock.